client/src/containers/HomePage/HomePage.jsx

```diff
@@ -199,16 +200,30 @@ class HomePage extends Component {
       && !this.chartsRef.current.state.dialogOpen
     ) {
       // once you click "next" when the table is open, it will throw an error because the current target (reports) is technically closed
-      // the "step-after" part of the event can't mount because the target isn't found -> need to make it so that it manually skips over?
       this.setState({ stepIndex: index + 1 });
-    } else if (
+    } else if (index == 7 && [EVENTS.STEP_AFTER].includes(type)) {
+      const { current = {} } = this.mapRef;
+      const { leafletElement: map } = current;
+      map.flyTo(MAP_LOCATIONS.losAngelesCountyCenter, 6 /* ZOOM */, {
+        duration: 0.5,
+      });
+      setTimeout(() => { // Wait for map to pan + zoom to la county then move to next step so target can lock
+        this.setState({ stepIndex: index + (action === ACTIONS.PREV ? -1 : 1) });
+      }, 500);
+    }
+    else if (
       [EVENTS.STEP_AFTER, EVENTS.TARGET_NOT_FOUND].includes(type)
       && index != 5
     ) {
       // Update state to advance the tour
       this.setState({ stepIndex: index + (action === ACTIONS.PREV ? -1 : 1) });
     } else if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
       // Need to set our running state to false, so we can restart if we click start again.
+      const { current = {} } = this.mapRef;
+      const { leafletElement: map } = current;
+      map.flyTo(MAP_LOCATIONS.usaCenter, 4, { // pan + zoom out back to usa center when tutorial is done
+        duration: 0.5,
+      });
```

Comment:
Are there any edge cases where this might be an issue of zooming?

Reply:
E.g. on different screens, do they take different times?

Reply:
After adding an extra step to make the user zoom out of the map after viewing it at a county level, I don't think this "flyto" usage is needed anymore, just going to remove it to avoid any bugs with it